scripts/wsrep_sst_mariabackup.sh

```diff
@@ -522,9 +524,13 @@ wait_for_listen()
     local PORT=$1
     local ADDR=$2
     local MODULE=$3
+    if ! which lsof > /dev/null; then
+       wsrep_log_error "lsof tool not found in PATH! Make sure you have it installed."
+       exit 1 
+    fi
     for i in {1..50}
     do
-        ss -p state listening "( sport = :$PORT )" | grep -qE 'socat|nc' && break
+        lsof -i :$PORT | grep LISTEN | grep -qE 'socat|nc' && break
```

Comment:
Or more concisely like: https://github.com/MariaDB/server/blob/10.2/scripts/wsrep_sst_xtrabackup-v2.sh#L660

Reply:
As wait_for_listen is reused in multiple scripts, it could go into wsrep_sst_common.sh... Then `ss`, `lsof`, `sockstat` checks can be performed centrally.